thrift/src/main/thrift/atoms/guide.thrift

```diff
@@ -0,0 +1,19 @@
+namespace * contentatom.guide
+namespace java com.gu.contentatom.thrift.atom.guide
+#@namespace scala com.gu.contentatom.thrift.atom.guide
+
+include "../shared.thrift"
+include "entity.thrift"
+
+struct GuideAtom {
+  1: optional string typeLabel
+  2: optional list<string> tags
```

Comment:
Yes I think this is a problem in the editing UI. I would remove the tag  fields in this case and if we need to do something like what you've said we'll use labels. If labels aren't working correctly in the atom-workshop tool then we should ask for support on that.